spec/categorical_spec.rb

```diff
@@ -0,0 +1,102 @@
+describe Daru::Vector do
+  context "initialize" do
+    let(:dv) { Daru::Vector.new [:a, 1, :a, 1, :c], type: :category }
+    subject { dv }
+    
+    it { is_expected.to be_a Daru::Vector }
+    its(:size) { is_expected.to eq 5 }
+    its(:type) { is_expected.to eq :category }
```

Comment:
Oh, if other tools use `category`, then it makes sense to stick to it here of course.

I see that pandas indeed uses dtype `category` (though their [help page](http://pandas.pydata.org/pandas-docs/stable/categorical.html) is titled "categorical"). Does type in daru play the same role as dtype in pandas?